htmresearch/frameworks/nlp/classify_htm.py

```diff
@@ -84,6 +84,10 @@ def __init__(self,
       self.networkConfig["classifierRegionConfig"].get("regionName")]
 
 
+  def getKNN(self):
+    return self.classifierRegion.getSelf()._knn
```

Comment:
@subutai Fixed. I'm curious where this method is defined. I can't seem to find it in the codebase.